demos/python_demos/formula_recognition_demo/formula_recognition_demo.py

```diff
@@ -100,10 +162,16 @@ def build_argparser():
     args.add_argument("--max_formula_len",
                       help="Optional. Defines maximum length of the formula (number of tokens to decode)",
                       default="128", type=int)
+    args.add_argument("--conf_thresh", help="Optional. Probability threshold to trat model prediction as meaningful",
+                      default=CONFIDENCE_THRESH, type=float)
```

Comment:
```suggestion
                      default=0.95, type=float)
```
So you can remove `CONFIDENCE_THRESH`.

Reply:
Removed this